mobile/lib/firebase_options.dart

```diff
@@ -25,7 +29,13 @@ class DefaultFirebaseOptions {
       case TargetPlatform.android:
         return android;
       case TargetPlatform.iOS:
-        return ios;
+        switch (network) {
+          case "regtest":
+            return iosTest;
+          case "mainnet":
```

Comment:
🤔 is this a fall through to `iosProd`? Might be clearer if we'd return the `iosProd` also on "mainnet"

Reply:
Yes, saving 1 line of code :)